packages/protocol/test-sol/devchain/Import05Dependencies.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.5.13;
```

Comment:
Could this be merged with the `./migration/05Links.sol`?

Reply:
Unfortunately it cannot since it needs to be in e2e folder